src/ImageRunModal.scss

```diff
@@ -0,0 +1,19 @@
+@import '@patternfly/patternfly/layouts/Grid/grid.css';
+// Note: PatternFly's "Flex" utilities also apply to grid alignment
+@import "@patternfly/patternfly/utilities/Flex/flex.css";
```

Comment:
This is not needed. You use directly the React component. when you import the React component the CSS of it is autoimported.

Reply:
Fixed